Integrations/integration-HybridAnalysis.yml

```diff
@@ -373,211 +562,430 @@ script:
-        - contextPath: File.total_network_connections
-          description: Total network connections of the file
-          type: string
-        - contextPath: File.total_processes
-          description: Total processes count of the file
-          type: string
-        - contextPath: File.total_signatures
-          description: Total signatures count if the file
-          type: string
-        - contextPath: File.hosts
-          description: List of file's hosts
-          type: string
-        - contextPath: File.isinteresting
-          description: If server found this file interesting
-          type: string
-        - contextPath: File.domains
-          description: List of file's related domains
-          type: string
-        - contextPath: File.isurlanalysis
-          description: If file analyzed by url
-          type: string
-        - contextPath: File.Malicious.Vendor
-          description: or malicious files, the vendor that made the decision
-          type: string
-        - contextPath: File.Malicious.Description
-          description: For malicious files, the reason for the vendor to make the decision
-          type: string
-      description: Get summary information for a given MD5, SHA1 or SHA256 and all the
-        reports generated for any environment ID
-    - name: hybrid-analysis-submit-sample
-      arguments:
-        - name: entryId
-          required: true
-          default: true
-          description: War-room entry ID of sample file
-        - name: environmentID
-          required: true
-          description: Environment ID to submit file to (get all IDs via vx-get-environments)
-      outputs:
-        - contextPath: HybridAnalysis.Submit.JobID
-          description: JobID of the submission
-          type: string
-        - contextPath: HybridAnalysis.Submit.SHA256
-          description: 'SHA of the submission '
-          type: string
-        - contextPath: HybridAnalysis.Submit.EnvironmentID
-          description: 'EnvironmentID of the submission '
-          type: string
-      description: Submit a file from investigation to analysis server  (NOTE - minimum
-        required authorization is `default`)
-    - name: hybrid-analysis-search
-      arguments:
-        - name: query
-          description: Hybrid Analysis query syntax (see `<server url>/faq#advanced-search-options`
-            for more details). examples -  url:google, host:95.181.53.78
-        - name: filename
-          description: Filename e.g. invoice.exe
-        - name: filetype
-          description: Filetype e.g. docx
-        - name: filetype_desc
-          description: Filetype description e.g. PE32 executable
-        - name: env_id
-          description: Environment Id
-        - name: country
-          description: Country (3 digit ISO) e.g. swe
-        - name: verdict
-          description: 'Verdict e.g. 1 (available: 1 ''whitelisted’, 2 ''no verdict’,
+  - name: hybrid-analysis-scan
```

Comment:
is there a reason all the lines below this one (including this one) used to be tabbed out and now they aren't?

Reply:
@reutshal had a chance to look at this?

Reply:
@reutshal Upload the yml yo demisto to make sure we are good here.

Reply:
Looking good